runtime/v2/binary.go

```diff
@@ -103,7 +103,7 @@ func (b *binary) Start(ctx context.Context, opts *types.Any, onClose func()) (_
 			log.G(ctx).WithError(err).Error("copy shim log")
 		}
 	}()
-	out, err := cmd.CombinedOutput()
+	out, err := cmd.Output()
```

Comment:
I think in some cases we may want the err stream for the below error check.

Reply:
IIUC, the binary expects a single line with the address.
If shim runs into an error, that info will be kept in the `error` structure, not in stderr.
So, we should not miss any important information.

Reply:
https://golang.org/pkg/os/exec/#Cmd.Output

>  Output runs the command and returns its standard output. Any returned error will usually be of type *ExitError. If c.Stderr was nil, Output populates ExitError.Stderr.

I agree with @dmitsh. This change wouldn't lose stderr. Mixing stdout and stderr doesn't make much sense here I think.

Reply:
> This change wouldn't lose stderr.

@kzys `ExitError` will not include `Stderr` data in it's error string output: https://github.com/golang/go/blob/master/src/os/exec/exec.go#L478
So if we go this way, we should check whether returned error is `ExitError` and read `Stderr` field.